Containerfile.tests

```diff
@@ -0,0 +1,30 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+FROM fedora:34
```

Comment:
This can be `FROM tokman`, this is how the image built by `make build` is going to be called. That one will have all the service dependencies installed, you would need to install only the deps required for the tests (make, flexmock, so on).

Reply:
Using "FROM tokman" in Containerfile.tests =>

/usr/bin/podman build -f Containerfile.tests -t tokman-test .
STEP 1: FROM tokman
? Please select an image: 
  ▸ registry.fedoraproject.org/tokman:latest
    registry.access.redhat.com/tokman:latest
    registry.centos.org/tokman:latest
    docker.io/library/tokman:latest

So I think this is not quite right yet.

Reply:
After updating Containerfile.tests to look like:

```
# Copyright Contributors to the Packit project.
# SPDX-License-Identifier: MIT

FROM tokman

RUN dnf install -y \
    findutils \
    make \
    python3-flexmock \
    python3-pytest \
    && dnf clean all

WORKDIR /src
```

I first ran `make build`, which built the `tokman` image on localhost, and then `make build-test-image`, which used the previous image, installed the additional dependencies required by the tests and changeed the working directory to `/src`. I was able to run `make check-in-container` after this and see the tests passing.

Having `build-test-image` depend on `build` would have required running only `make build-test-image` for building. See my comment on this further below.